src/pages/registration/contributors_tab/components/AffiliationsCell.tsx

```diff
@@ -58,8 +58,12 @@ export const AffiliationsCell = ({ affiliations = [], authorName, baseFieldName
           {affiliation.type === 'Organization' && (
             <OrganizationBox
               unitUri={affiliation.id}
+              authorName={authorName}
+              affiliations={affiliations}
+              baseFieldName={baseFieldName}
               removeAffiliation={() => removeAffiliation(index)}
               sx={{ width: '100%' }}
+              canEdit
```

Comment:
ikke sjekket nøye, så et åpent spørsmål:
Er dette implisitt satt av andre props, feks `baseFieldName`?
Altså finnes det en case hvor man sender inn `baseFieldName`, men man _ikke_ skal kunne endre?
Hvis ikke er det kanskje trygger å ha `const canEdit = !!baseFieldName` e.l. i `OrganizationBox`?

Reply:
Ja det er ihvertfall andre felter som er obligatoriske for å kunne editere. Men da risikerer vi at edit-ikonet kommer opp et sted man ikke ønsker det bare fordi man sendte inn baseFieldName? Jeg føler det blir litt mer eksplisitt valg med en sånn boolean, selv om jeg er enig i at den er der nesten bare for pynt.

Reply:
Kan gått være det trengs ja, så helt fint det!
Har bare merket flere ganger selv at jeg har lagt til flere props bare for å åpne for flere bugs 😅 
Typisk når det er noe som bare skal kunne slettes under gitte forutsetninger, så er det lett å legge inn props som dette
```
interface ComponentProps {
  onDelete?: () => void;
  canDelete: boolean;
}
```
Men det åpner bare for å legge inn rare bugs men motsridende props som feks `<ComponentProps canDelete />`, og liker da heller å bare basere det på `const canDelete = !!onDelete` i stedet. Men det forutsetter på at det finnes en logisk prop for det ja som du sier, ellers blir det bare mystisk magi